### PR TITLE
#161699088 Fix trip planner page refresh

### PR DESCRIPTION
**Pull request by Nedson202**
#### What does this PR do?
Updates the trip planner page automatically after adding, deleting, or editing a checklist item

#### Description of Task to be completed?
- remove the line of code that fetches checklist items after adding, deleting or editing an item
- update create, edit, and delete reducer to hold new changes and automatically update the page
- add tests for new changes

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Switch to the project directory - cd travel_tool_front
- Checkout - git checkout ch-fix-trip-planner-page-refresh-161699088
- Start the app - make start
- Start the backend of this project - git clone https://github.com/andela/travel_tool_back.git && make start
- Navigate to travela in your browser and log in
- Update your role on the database to a travel admin
- Navigate to trip planner page
- Add an item, Edit an Item, Delete an item

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161699088](https://www.pivotaltracker.com/story/show/161699088)

#### Screenshots (if appropriate)
trip-planner reload

#### Questions:

### CODE REVIEW

File: src/views/Checklist/index.scss [hunk 1]
```diff
@@ -217,4 +217,4 @@
       cursor: pointer;
     }
   }
-}
+}
```
Comment (ssewilliam) [inline src/views/Checklist/index.scss]: Please add a new line

Comment (Nedson202) [reply src/views/Checklist/index.scss]: Done @ssewilliam

**Comment by babadee001** [comment]
@Nedson202 There are some code climate issues on the PR. Can this be checked?

**Comment by Nedson202** [comment]
Two out of the three issues have to do with `Proptypes Validation` which is a common issue on many PRs flagged by code-climate. A refactoring of the total number of functions was done by @ssewilliam's on his current PR regarding `restoring deleted checklist item`. Having to do likewise here would be me duplicating efforts and also cause conflicts with @ssewilliam's work. I am in sync with @ssewilliam on this.

**Pull request closed without merging**